index.html

```diff
@@ -230,19 +224,19 @@ <h3>Locales and Internationalization</h3>
         </ul>
         ... and many more. </p>
         
-        <p class="definition"><dfn id="internationalization" data-lt="internationalization|I18N|internationalized">Internationalization</dfn>. The design and development of a product that is enabled for target audiences that vary in culture, region, or language. Internationalization is sometimes abbreviated <code>I18N</code> because there are eighteen letters between the "i" and the "n" in the English word.</p>
+        <p class="definition"><dfn id="internationalization" data-lt="internationalization|I18N|internationalized">Internationalization</dfn>. The design and development of a product that is enabled for target audiences that vary in culture, region, or language. Internationalization is sometimes abbreviated <code>i18n</code> because there are eighteen letters between the "I" and the "N" in the English word.</p>
```

Comment:
Ooooh! `i18n` vs. `I18N` vs. `i18N` !! Let the battle begin!

I agree with the capitalization in the "between the 'I' and the 'N'" part. I happen to prefer shouting "I18N" personally. I had hoped to avoid discussion of capitalization (and in my conference tutorial I always have it on exactly one slide--where I explain what it is and why I don't prefer to use it most of the time).

The problem here isn't this presentation, but the later one of l10n, because people rightly note that the 'l' can look like an I and the darned numeronyms are impossible to remember in the first place.

Reply:
Yeah, i only changed it because these are not acronyms, and we treat them as ordinary words in other places where we use the terms in the WG outputs.

Reply:
LOL. They're not acronyms--they're something else entirely 🙈. Again, my personal preference is for I18N, but I note that the WG has a long-standing established style. This isn't the place to change it. You can ignore this comment and the following one.